### PR TITLE
Disable Kubescape security scanner in e2e tests

### PR DESCRIPTION
**Pull request by mboersma**
**What type of PR is this?**

/kind failing-test

**What this PR does / why we need it**:

The optional Kubescape scanner test has somehow begun failing, even though we pinned it at a known working version. Since this is purely an informative test spec, let's disable it so it doesn't block PRs in e2e.

**Which issue(s) this PR fixes**:

n/a

**Special notes for your reviewer**:

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by jackfrancis — APPROVED** [review]
/lgtm
/approve

**Comment by mboersma** [comment]
/hold

This may be skipping the entire VMSS `It` spec according to the logs. Let me find another way.

File: test/e2e/azure_test.go
Comment (jackfrancis) [inline test/e2e/azure_test.go:391]: Let's just remove this entire `By` block. git history (and the `KubescapeSpec` func) will allow us to easily retrieve it.

Comment (jackfrancis) [reply test/e2e/azure_test.go:391]: I was gonna say let's just merge this as-is so we can unblock all the PRs in the queue but we have an Azure bastion flake in this test run so yeah, go ahead and delete this whole test section from the VMSS test flow and let's try again!

**Comment by CecileRobertMichon** [comment]
Let's also file an issue to follow up on this a re-evaluate whether we want this test at all (and remove the spec if not, or if do to fix it and re-enable it).

**Comment by mboersma** [comment]
See #2770.

**Comment by jackfrancis** [comment]
` msg: "Azure Bastion provisioning failed with state: \"Failed\"",`

Where is the aneurism emoji?

**Comment by CecileRobertMichon** [comment]
/override pull-cluster-api-provider-azure-e2e

the test failure above is due to a known Azure outage, #2772 is disabling the test for now.

**Pull request merged**
